roles/docker/tasks/cleanup.yml

```diff
@@ -12,3 +12,9 @@
   become: true
   with_items:
     - "{{ docker_api_port }}"
+
+- name: cleanup contiv-compose
```

Comment:
> would it still not require a download to check the version?

yes, the check is there. It is broken as @joji pointed out but it is there and can be fixed.

> I am thinking then that we also need a release schedule to put the versioning on the image names?

no, we don't need to. We check for output of `contiv-compose --version` and match it to desired version. It will only install if the version out doesn't match the expected output.